content/commcalls/2019-03-27.md

```diff
@@ -0,0 +1,50 @@
+---
+title: 'Research Applications of rOpenSci Taxonomy and Biodiversity Tools'
+dateStart: '2019-03-27T14:00:00'
+dateEnd: '2019-03-27T15:00:00'
+date: '2019-03-27'
+timezone: America/Los_Angeles
+blog: https://ropensci.org/blog/2018/12/05/commcall-dec2018/
+description: 'Helping people learn about rOpenSci’s packages to access and analyze taxonomy and biodiversity data, and to recognize the breadth and depth of their applications. Presentations will start with an introduction and details on some specific packages and we’ll hear from several people about their “use cases in the wild”.'
+authors:
+  - Zachary Foster
+  - Scott Chamberlain
+  - Margaret Siple
+  - Kathryn Turner
+  - Kari Norman
+location: Community call (teleconference)
+country: "\U0001F310"
+attendees: All are welcome
+deets: 'Join from PC, Mac, Linux, iOS or Android: https://berkeley.zoom.us/j/105842728 (requires app installation first time)<br/>
+
+One tap mobile<br/>  
+  +16699006833,,105842728# US (San Jose)<br/>  
+  +16465588656,,105842728# US (New York)<br/>  
+
+Dial by your location<br/>
++1 669 900 6833 US (San Jose)<br/>
++1 646 558 8656 US (New York)<br/>
+877 369 0926 US Toll-free<br/>
+877 853 5247 US Toll-free<br/>
+
+Meeting ID: 105 842 728<br/>
+Find your local number: https://zoom.us/u/abVn1J8HqM
+'
+tags:
+  - taxonomy
+  - community call
+  - biodiversity
+  - taxize
+  - taxizedb
+  - rgbif
+  - spocc
+  - taxa
+  - metacoder
+  - worrms
+  - wikitaxa
+  - taxview
+  - ritis
+  - zbank
+---
+NA
+---
```

Comment:
```suggestion
```

you can remove these two last lines, they appeared at some point in the files due to my YAML munging and my not paying enough attention. :-)